packages/traceloop-sdk/traceloop/sdk/tracing/tracing.py

```diff
@@ -73,6 +73,7 @@ def __new__(
         instruments: Optional[Set[Instruments]] = None,
         block_instruments: Optional[Set[Instruments]] = None,
         image_uploader: ImageUploader = None,
+        on_end: Optional[Callable[["ReadableSpan"], None]] = None,
```

Comment:
I'd use a more informative parameter name here than `on_end`. Maybe `span_postprocess_callback` or `before_span_export` or something like that.